src/style.css

```
@@ -1,3 +1,110 @@
+:root {
+  --font-text: "Roboto", sans-serif;
+  --font-size-small: 12px;
+  --bg-color-white: #fff;
+  --text-color-black: #000;
+  --text-color-blue: #00acdc;
+
+}
+
 body {
   margin: 0;
+  font-family: var(--font-text);
+  font-size: var(--font-size-small);
+  font-weight: 500;
+  color: var(--text-color-black);
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  width: 100%;
+  max-width: 1200px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  box-sizing: border-box;
+}
+
+.header__logo--link {
+  object-fit: contain;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  margin: 0 0 0 20px;
+  text-transform: uppercase;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__item--link {
+  display: inline-block;
+  height: 60px;
+  line-height: 60px;
+  text-decoration: none;
+}
+
+.nav__item--link:link {
+  color: var(--text-color-black);
+}
+
+.nav__item--link:visited {
+  color: var(--text-color-blue);
+}
+
+.nav__item--link:hover {
+  color: var(--text-color-blue);
+}
```

```suggestion
.nav__item--link:link,
.nav__item--link:visited,
.nav__item--link:hover {
  color: var(--text-color-blue);
}
```
You can combine selectors

---

combined